include/envoy/network/connection.h

```diff
@@ -32,6 +32,12 @@ enum class ConnectionEvent {
  */
 enum class ConnectionBufferType { Read, Write };
 
+/**
+ * Callback function for when bytes have been sent by a connection.
+ * @param bytes_sent supplies the number of bytes written to the connection.
+ */
+typedef std::function<void(uint64_t bytes_sent)> BytesSentCb;
```

Comment:
Should this be here, or in class Connection?

Reply:
Doesn't really matter to me. The other callback is at Network namespace level. If I were to do it again I would probably put it inside the Connection class or in a Connection namespace. So I guess mild preference to move this inside Connection class, but whatever you want is fine.